java/code/src/com/redhat/rhn/frontend/strings/java/StringResource_en_US.xml

```diff
@@ -8385,7 +8385,7 @@ Thank you for using @@PRODUCT_NAME@@.
         <source>Select first character</source>
       </trans-unit>
       <trans-unit id="login.jsp.satbody2">
-        <source>For details on @@PRODUCT_NAME@@, please visit our website &lt;a href=&quot;http://www.suse.com/products/suse-manager/&quot; target=&quot;_new&quot;&gt;&lt;i class=&quot;fa fa-globe&quot;&gt;&lt;/i&gt;&lt;/a&gt;. </source>
+        <source>For details on @@PRODUCT_NAME@@, please visit our website &lt;a href="http://www.suse.com/products/suse-manager/" target="_new"&gt;&lt;i class="fa fa-globe"&gt;&lt;/i&gt;&lt;/a&gt;. </source>
```

Comment:
While this code predates these changes, `target="_new"` has not been a part of the HTML spec for quite a long time now. Most browsers support it for legacy purposes, but it would be better to have `target="_blank" rel="noreferrer noopener"`. The latter addition is a security consideration, please see [the related MDN article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target) for more details.

Reply:
I think this should be covered in an extra PR. This PR is only about the use of the new xliffmerger tool and the
changes resulting out if it.
A new PR which fix all of this is very welcome